src/main/java/com/intel/genomicsdb/model/CommandLineImportConfig.java

```diff
@@ -19,26 +20,41 @@
 
 import static java.util.stream.Collectors.toList;
 
-public class CommandLineImportConfig extends BaseImportConfig {
+public class CommandLineImportConfig extends ParallelImportConfig {
+    GenomicsDBImportConfiguration.Partition.Builder partitionBuilder =
+            GenomicsDBImportConfiguration.Partition.newBuilder();
+    GenomicsDBImportConfiguration.GATK4Integration.Builder gatk4IntegrationBuilder =
+            GenomicsDBImportConfiguration.GATK4Integration.newBuilder();
+    GenomicsDBImportConfiguration.ImportConfiguration.Builder configurationBuilder =
+            GenomicsDBImportConfiguration.ImportConfiguration.newBuilder();
 
     public CommandLineImportConfig(final String command, final String[] commandArgs) {
         Getopt getOpt = new Getopt(command, commandArgs, "w:A:L:", resolveLongOpt());
         resolveCommandArgs(getOpt);
+        try {
+            configurationBuilder.addColumnPartitions(partitionBuilder.build());
+            configurationBuilder.setGatk4IntegrationParameters(gatk4IntegrationBuilder.build());
+            this.setImportConfiguration(configurationBuilder.build());
+        } catch (UninitializedMessageException ex) {
+            throw new IllegalArgumentException("Invalid usage. Correct way of using arguments: -L chromosome:interval " +
```

Comment:
This error msg duplication I'm going to refactor in the next PR - removing duplications.